dvc/tree/http.py

```diff
@@ -136,12 +136,6 @@ def get_file_hash(self, path_info):
                 "Content-MD5 header for '{url}'".format(url=url)
             )
 
-        if etag.startswith("W/"):
-            raise DvcException(
-                "Weak ETags are not supported."
-                " (Etag: '{etag}', URL: '{url}')".format(etag=etag, url=url)
-            )
-
```

Comment:
@pared, what about imports?

Reply:
Yeah, thats right, I changed the approach.